App.js

```diff
@@ -22,19 +22,34 @@ export default class App extends Component<{}> {
     usabilla.setFormDidFailLoading((reminder) => console.log("Error loading form:", reminder))
     usabilla.setFormDidClose((reminder) => console.log(reminder.formId))
   }
+
   onFormLoaded() {
     usabilla.showLoadedForm()
   }
+
   requestForm() {
     usabilla.loadFeedbackForm("YOUR_FORM_ID_HERE")
   }
+
+  resetCampaignData() {
+    usabilla.resetCampaignData()
+  }
+
+  sendEvent() {
+    usabilla.sendEvent("YOUR_EVENT_HERE")
```

Comment:
Can we add a text field to the view that can allow us to send different events each time?